[Sun Jun 07 20:48:51.091423 2020] [mpm_winnt:notice] [pid 12880:tid 796] AH00455: Apache/2.4.6 (Win32) PHP/5.4.17 configured -- resuming normal operations
[Sun Jun 07 20:48:51.092423 2020] [mpm_winnt:notice] [pid 12880:tid 796] AH00456: Apache Lounge VC10 Server built: Jul 16 2013 11:15:41
[Sun Jun 07 20:48:51.092423 2020] [core:notice] [pid 12880:tid 796] AH00094: Command line: 'C:\\Users\\andre\\Documents\\Projetos\\Atividades\\Keyar\\Website/apache2/bin/httpd_usbwv8.exe -d C:/Users/andre/Documents/Projetos/Atividades/Keyar/Website/apache2'
[Sun Jun 07 20:48:51.105415 2020] [mpm_winnt:notice] [pid 12880:tid 796] AH00418: Parent: Created child process 7628
[Sun Jun 07 20:48:54.092578 2020] [mpm_winnt:notice] [pid 7628:tid 920] AH00354: Child: Starting 64 worker threads.
[Sun Jun 07 20:49:04.366977 2020] [:error] [pid 7628:tid 1332] [client 127.0.0.1:2175] PHP Fatal error:  Uncaught exception 'PDOException' with message 'SQLSTATE[HY000] [1045] Access denied for user 'u208484937_keyar'@'localhost' (using password: YES)' in C:\\Users\\andre\\Documents\\Projetos\\Atividades\\Keyar\\Website\\root\\src\\base\\TGerenciadorConexaoBD.php:26\nStack trace:\n#0 C:\\Users\\andre\\Documents\\Projetos\\Atividades\\Keyar\\Website\\root\\src\\base\\TGerenciadorConexaoBD.php(26): PDO->__construct('mysql:host=loca...', 'u208484937_keya...', 'keyar_keyar@123')\n#1 C:\\Users\\andre\\Documents\\Projetos\\Atividades\\Keyar\\Website\\root\\src\\autenticacao\\controller\\CUsuario.php(17): MUsuario->__construct()\n#2 C:\\Users\\andre\\Documents\\Projetos\\Atividades\\Keyar\\Website\\root\\src\\autenticacao\\view\\VUsuario.php(5): CUsuario->__construct()\n#3 C:\\Users\\andre\\Documents\\Projetos\\Atividades\\Keyar\\Website\\root\\public\\index.php(2): include('C:\\Users\\andre\\...')\n#4 {main}\n  thrown in C:\\Users\\andre\\Documents\\Projetos\\Atividades\\Keyar\\Website\\root\\src\\base\\TGerenciadorConexaoBD.php on line 26
[Sun Jun 07 21:10:43.628173 2020] [:error] [pid 7628:tid 1332] [client 127.0.0.1:2882] PHP Fatal error:  Uncaught exception 'PDOException' with message 'There is no active transaction' in C:\\Users\\andre\\Documents\\Projetos\\Atividades\\Keyar\\Website\\root\\src\\base\\TGerenciadorConexaoBD.php:69\nStack trace:\n#0 C:\\Users\\andre\\Documents\\Projetos\\Atividades\\Keyar\\Website\\root\\src\\base\\TGerenciadorConexaoBD.php(69): PDO->rollBack()\n#1 C:\\Users\\andre\\Documents\\Projetos\\Atividades\\Keyar\\Website\\root\\src\\autenticacao\\controller\\CUsuario.php(115): MUsuario->rollback()\n#2 C:\\Users\\andre\\Documents\\Projetos\\Atividades\\Keyar\\Website\\root\\src\\autenticacao\\controller\\CUsuario.php(26): CUsuario->efetuarLogin(Array)\n#3 C:\\Users\\andre\\Documents\\Projetos\\Atividades\\Keyar\\Website\\root\\src\\autenticacao\\view\\VUsuario.php(8): CUsuario->executarAcao('efetuarLogin', Array)\n#4 C:\\Users\\andre\\Documents\\Projetos\\Atividades\\Keyar\\Website\\root\\public\\index.php(2): include('C:\\Users\\andre\\...')\n#5 {main}\n  thrown in C:\\Users\\andre\\Documents\\Projetos\\Atividades\\Keyar\\Website\\root\\src\\base\\TGerenciadorConexaoBD.php on line 69, referer: http://127.0.0.1:8080/public/index.php
